test/e2e/apps/statefulset.go

```diff
@@ -1130,9 +1127,10 @@ func rollbackTest(c clientset.Interface, ns string, ss *appsv1.StatefulSet) {
 	framework.ExpectNoError(err)
 	ss = e2esset.WaitForStatus(c, ss)
 	currentRevision, updateRevision = ss.Status.CurrentRevision, ss.Status.UpdateRevision
-	gomega.Expect(currentRevision).NotTo(gomega.Equal(updateRevision),
-		"Current revision should not equal update revision during roll back")
 	framework.ExpectEqual(priorRevision, updateRevision, "Prior revision should equal update revision during roll back")
+	framework.ExpectNotEqual(currentRevision, updateRevision, "Current revision should not equal update revision during roll back")
+	gomega.Expect(priorRevision).To(gomega.Equal(updateRevision),
```

Comment:
why this change?

Reply:
looks like this is same as line 1130 right?

Reply:
Nice catch, line 1132 should be removed I think also.